projects.html

```diff
@@ -16,7 +16,32 @@
                 <div id="profileDiv" class="profile-div"></div>
             </div>
             <div id="mainContentContainer" class="main-content-container">
-                <div id="navbar" class="navbar-element"></div>
+                <div id="navbar" class="navbar-element">
+                    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+                      <div class="container-fluid">
+                        <a class="navbar-brand" href="#">Gitsub</a>
+                        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                          <span class="navbar-toggler-icon"></span>
+                        </button>
+                        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                            <li class="nav-item">
+                              <a class="nav-link" href="index.html">Overview</a>
+                            </li>
+                            <li class="nav-item">
+                              <a class="nav-link" href="repositories.html">Repositories</a>
+                            </li>
+                            <li class="nav-item">
```

Comment:
Switch the order of this one and the Projects. Order of the Navabr should be Overview, Repositories, Projects, and Packages.